cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -133,38 +145,33 @@ def test_get_course_list(self):
         # check both course lists have same courses
         self.assertEqual(courses_list, courses_list_by_groups)
 
-    def test_get_course_list_when_ccx(self):
+    def test_courses_list_with_ccx_courses(self):
         """
-        Assert that courses with CCXLocator are filter in course listing.
+        Tests that CCX courses are filtered in course listing.
         """
+        # Create a course and assign access roles to user.
         course_location = self.store.make_course_key('Org1', 'Course1', 'Run1')
-        self._create_course_with_access_groups(course_location, self.user)
+        course = self._create_course_with_access_groups(course_location, self.user)
 
-        # get courses through iterating all courses
-        courses_list, __ = _accessible_courses_list(self.request)
-        self.assertEqual(len(courses_list), 1)
+        # Create a ccx course key and add assign access roles to user.
+        ccx_course_key = CCXLocator.from_course_locator(course.id, u"1")
+        self._add_role_access_to_user(self.user, ccx_course_key)
 
-        # get courses by reversing group name formats
-        courses_list_by_groups, __ = _accessible_courses_list_from_groups(self.request)
-        self.assertEqual(len(courses_list_by_groups), 1)
+        # Test that CCX courses are filtered out.
+        courses_list, __ = _accessible_courses_list_from_groups(self.request)
+        self.assertEqual(len(courses_list), 1)
+        self.assertEqual(unicode(courses_list[0].id), unicode(course.id))
 
-        # assert no course in listing with ccx id
-        ccx_course = Mock()
-        course_key = CourseKey.from_string('course-v1:FakeOrg+CN1+CR-FALLNEVER1')
-        ccx_course.id = CCXLocator.from_course_locator(course_key, u"1")
-
-        with patch(
-            'xmodule.modulestore.mixed.MixedModuleStore.get_course',
-            return_value=ccx_course
-        ), patch(
-            'xmodule.modulestore.mixed.MixedModuleStore.get_courses',
-            Mock(return_value=[ccx_course])
-        ):
-            courses_list, __ = _accessible_courses_list_from_groups(self.request)
-            self.assertEqual(len(courses_list), 0)
+        # Get all courses which user has access.
+        instructor_courses = UserBasedRole(self.user, CourseInstructorRole.ROLE).courses_with_role()
+        staff_courses = UserBasedRole(self.user, CourseStaffRole.ROLE).courses_with_role()
+        all_courses = (instructor_courses | staff_courses)
 
-            courses_list, __ = _accessible_courses_list(self.request)
```

Comment:
@awaisdar001 where is test for https://github.com/edx/edx-platform/blob/aj/fix-cms-dashboard-for-ccx/cms/djangoapps/contentstore/views/course.py#L395 gone?